src/pages/tx/_components/GasDetails.tsx

```diff
@@ -0,0 +1,206 @@
+import useWindowDimensions from "hooks/useWindowDimensions";
```

Comment:
Spacing between figma design and actual on gas detail section (desktop view)
![image](https://github.com/WavesHQ/metascan/assets/13292662/3f216e36-661b-4986-9e59-5b26210992bd)
![image](https://github.com/WavesHQ/metascan/assets/13292662/82b95d7e-64c5-4d76-ac01-b1f86ac16c66)

Reply:
weird, just checked both on local and deploy preview and it looks fine for me 😵 
<img width="1010" alt="image" src="https://github.com/WavesHQ/metascan/assets/17063281/d7d60d65-4535-4e25-9fd6-a517dd719293">